dev_config/src/admin_config/profiles/Admin.profile

```diff
@@ -62,6 +62,10 @@
         <tab>%%%NAMESPACE%%%Affiliation__c</tab>
         <visibility>DefaultOn</visibility>
     </tabVisibilities>
+    <tabVisibilities>
```

Comment:
@lifewithryan @xinnanlisf You both might already be aware of this, but this is what I was talking about the other day during standup where I couldn't figure out why I had no access to the new tabs when I spun up the scratch org.

The trick is to edit this file and add the tab visibilities for the Admin profile. This gets deployed to the scratch org in the deploy_dev_config task as part of the dev_org flow.